install_config/configuring_authentication.adoc

```diff
@@ -159,18 +159,24 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Configuring the ldap ca certificate <1>
 #openshift_master_ldap_ca=<ca text>
 # or
-#openshift_master_ldap_ca_file=<path to local ca file to use>
+#openshift_master_ldap_ca_file=<path to local ca file to use> <2>
 
 # Available variables for configuring certificates for other identity providers:
 #openshift_master_openid_ca
-#openshift_master_openid_ca_file
+#openshift_master_openid_ca_file <2>
 #openshift_master_request_header_ca
-#openshift_master_request_header_ca_file
+#openshift_master_request_header_ca_file <2>
 ----
 <1> If you specify your CA certificate location in the
```

Comment:
This is no longer correct, CA certificate location in `openshift_master_identity_providers` parameter is now ignored